app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt

```diff
@@ -28,35 +30,37 @@ import org.json.JSONObject
 
 abstract class EmbarkViewModel(
     private val tracker: EmbarkTracker,
-    private val valueStore: ValueStore,
+    private val valueStore: ValueStore
```

Comment:
I really like trailing commas, as they tend to produce cleaner diffs for reading when extending the code. I'm going to try to add a rule in `ktlint` to enforce it later on, but until then, could you enable them in your IDEs formatting rules? 😄